src/client/datascience/commands/serverSelector.ts

```diff
@@ -21,11 +22,15 @@ export class JupyterServerSelectorCommand implements IDisposable {
         this.disposables.push(
             this.commandManager.registerCommand(
                 Commands.SelectJupyterURI,
-                (local: boolean = true, userUri: string | undefined, source: 'nativeNotebookStatusBar' | 'commandPalette' | 'toolbar' = 'commandPalette') => {
+                (local: boolean = true, source: Uri | 'nativeNotebookStatusBar' | 'commandPalette' | 'toolbar' = 'commandPalette') => {
 
-                    if (!local && userUri) {
-                        traceInfo(`Setting Jupyter Server URI to remote: ${userUri}`);
-                        this.serverSelector.setJupyterURIToRemote(userUri)
+                    if (source instanceof Uri) {
+                        traceInfo(`Setting Jupyter Server URI to remote: ${source}`);
+                        console.log(`Setting Jupyter Server URI to remote: ${source}`)
```

Comment:
I assume you didn't mean to keep the console.log messages?

Reply:
Yeah, my bad! I removed them 😁